tls/s2n_connection.c

```diff
@@ -114,13 +68,11 @@ struct s2n_connection *s2n_connection_new(s2n_mode mode)
             GUARD_PTR(s2n_free(&blob));
             S2N_ERROR_PTR(S2N_ERR_CLIENT_MODE_DISABLED);
         }
-
-        conn->verify_cert_chain_cb = accept_all_rsa_certs;
+        conn->config = &s2n_unsafe_client_config;
```

Comment:
Could you add a comment above this that explains what is unsafe about this default config?